manifests/configmap.yaml

```diff
@@ -41,6 +41,7 @@ data:
   # enable_ebs_gp3_migration_max_size: "1000"
   # enable_init_containers: "true"
   # enable_lazy_spilo_upgrade: "false"
+  # enable_cross_namespace_secret: "false"
```

Comment:
again, alphabetical sorting. And it seems, the example is missing in the example `OperatorConfiguration` CRD manifest